### PR TITLE
Add `LaterPayClient.get_access_data()` and deprecate `.get_access()`

### PR DESCRIPTION
**Pull request by suligap**
Other new methods:
- `.get_request_headers()`
- `.get_access_url()`
- `.get_access_params()`

`.get_access_data()` uses `requests` library.

### CODE REVIEW

File: laterpay/__init__.py [hunk 1]
```diff
@@ -472,3 +482,78 @@ def get_access(self, article_ids, product_key=None):
...
+        """
+        if not isinstance(article_ids, (list, tuple)):
+            article_ids = [article_ids]
+
+        lptoken = str(lptoken or self.lptoken)
+
+        params = {
+            'cp': self.cp_key,
+            'ts': str(int(time.time())),
+            'lptoken': lptoken or self.lptoken,
```
Comment (slafs) [inline laterpay/__init__.py]: we already checked `lptoken` in `str` function few lines higher

File: laterpay/__init__.py [hunk 2]
```diff
@@ -472,3 +482,78 @@ def get_access(self, article_ids, product_key=None):
...
+            'article_id': article_ids,
+        }
+
+        params['hmac'] = signing.sign(
+            secret=self.shared_secret,
+            params=params.copy(),
+            url=self._access_url,
```
Comment (slafs) [inline laterpay/__init__.py]: Maybe we should use `.get_access_url()` here also? Or allow to pass that url as another argument in this method?

File: laterpay/__init__.py [hunk 3]
```diff
@@ -472,3 +482,78 @@ def get_access(self, article_ids, product_key=None):
...
         return data
+
+    def get_request_headers(self):
+        """
+        Return a ``dict`` of request headers to be sent to the API.
+        """
+        return {
+            'X-LP-APIVersion': 2,
+            # TODO: Automatic versioning.
+            'User-Agent': 'LaterPay Client - Python - v4.2.0'
```
Comment (slafs) [inline laterpay/__init__.py]: We should either mention this in "Release Checklist" in README or maybe try to use something like 
```
pkg_resources.get_distribution('laterpay-client').version
```

Comment (suligap) [inline laterpay/__init__.py]: Dropping the version info for now. This is out of scope of this PR.

File: laterpay/__init__.py (hunk 2 above)
Comment (suligap) [inline laterpay/__init__.py]: Thanks! Fixed.

File: laterpay/__init__.py (hunk 1 above)
Comment (suligap) [inline laterpay/__init__.py]: Thanks! Fixed.

Comment (slafs) [inline laterpay/__init__.py]: What I meant was that `'lptoken': lptoken,` is enough here ;). We already have `str(lptoken or self.lptoken)` 5 lines above this.

File: laterpay/__init__.py [hunk 4]
```diff
@@ -472,3 +482,78 @@ def get_access(self, article_ids, product_key=None):
...
+        """
+        if not isinstance(article_ids, (list, tuple)):
+            article_ids = [article_ids]
+
+        lptoken = str(lptoken or self.lptoken)
+
+        params = {
+            'cp': self.cp_key,
+            'ts': str(int(time.time())),
+            'lptoken': str(lptoken or self.lptoken),
```
Comment (slafs) [inline laterpay/__init__.py]: https://github.com/laterpay/laterpay-client-python/pull/62#discussion_r56489072

Comment (suligap) [inline laterpay/__init__.py]: I could swear I've removed 517.. thanks! :)

**Comment by slafs** [comment]
LGTM
